<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="b_sel_payload">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_payload">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Payload: </string>
    </property>
   </widget>
   <widget class="QLabel" name="l_payload_name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>none</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_programmer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Programmer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_prog_name">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>241</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_sel_progr">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_flash">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Flash</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_erase">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_probe">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Probe</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>426</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_sel_payload"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_sel_payload"/>
   <addaction name="act_extract"/>
   <addaction name="act_about"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act_sel_payload">
   <property name="text">
    <string>Select ROM</string>
   </property>
  </action>
  <action name="act_extract">
   <property name="text">
    <string>Extract</string>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
